buddyboss/buddyboss-platform
# fix sql query for mysql version 8.0.22

### All Submissions:


### Changes proposed in this Pull Request:
There is a mysql query in function `bp_document_get_root_parent_id` in the file `buddyboss-platform/bp-document/bp-document-functions.php` which is used to get the parent folder id of the uploaded file and that query showing different result for mysql latest version (the query return the correct folder ID for mysql version 5.7 but not for latest version). I updated the SQL query to make it work for both versions.

Fixes #2072

### How to test the changes in this Pull Request:

1. Go to user profile documents (activate media component and enable document upload for activity)
2. Create a folder and upload a file into that folder
3. Go to user profile activity feed page
4. See activity item is not broken now, before it was broken for MySQL version 8.0.22

### Proof Screenshots or Video
https://www.loom.com/share/57ea4859856b4c04a0421d092a2c6357

### Other information:
